### PR TITLE
chore: cleanup Team Replay feature flags

### PR DESCRIPTION
**Pull request by daibhin**
## Problem

Follow up from https://github.com/PostHog/posthog/pull/20539#discussion_r1501247297

## Changes

Remove unused / released feature flags for Team Replay

## How did you test this code?

No changes (unused code 🤞)

### CODE REVIEW

File: frontend/src/scenes/session-recordings/player/controller/PlayerSeekbarPreview.tsx [hunk 1]
```diff
@@ -76,9 +74,7 @@ export function PlayerSeekbarPreview({ minMs, maxMs, seekBarRef, activeMs }: Pla
 
     const isHovering = useIsHovering(seekBarRef)
 
-    const { featureFlags } = useValues(featureFlagLogic)
-    const alwaysShowSeekbarPreview = !!featureFlags[FEATURE_FLAGS.ALWAYS_SHOW_SEEKBAR_PREVIEW]
```
Comment (daibhin) [inline frontend/src/scenes/session-recordings/player/controller/PlayerSeekbarPreview.tsx:80]: Looks like this feature flag didn't exist in PostHog so this condition would always have been false. @pauldambra may remember what it was used for

File: frontend/src/lib/constants.tsx [hunk 2]
```diff
@@ -180,22 +179,17 @@ export const FEATURE_FLAGS = {
...
-    NOTEBOOK_CANVASES: 'notebook-canvases', // owner: #team-replay
     SESSION_RECORDING_SAMPLING: 'session-recording-sampling', // owner: #team-replay
     PERSON_FEED_CANVAS: 'person-feed-canvas', // owner: #project-canvas
     MULTI_PROJECT_FEATURE_FLAGS: 'multi-project-feature-flags', // owner: @jurajmajerik #team-feature-success
-    NETWORK_PAYLOAD_CAPTURE: 'network-payload-capture', // owner: #team-replay
     FEATURE_FLAG_COHORT_CREATION: 'feature-flag-cohort-creation', // owner: @neilkakkar #team-feature-success
     INSIGHT_HORIZONTAL_CONTROLS: 'insight-horizontal-controls', // owner: @benjackwhite
     SURVEYS_WIDGETS: 'surveys-widgets', // owner: @liyiy
     SCHEDULED_CHANGES_FEATURE_FLAGS: 'scheduled-changes-feature-flags', // owner: @jurajmajerik #team-feature-success
-    ALWAYS_SHOW_SEEKBAR_PREVIEW: 'always-show-seekbar-preview', // owner: #team-replay
```
Comment (pauldambra) [inline frontend/src/lib/constants.tsx:192]: this was more of a debug flag... so we could force it for users

I don't know if ever did though - just need to check it isn't on for anyone before we remove it 👍

**Review by pauldambra — APPROVED** [review]
👍

File: frontend/src/scenes/session-recordings/player/controller/PlayerSeekbarPreview.tsx (hunk 1 above)
Comment (pauldambra) [reply frontend/src/scenes/session-recordings/player/controller/PlayerSeekbarPreview.tsx:80]: Ah, we overlapped... 

if it doesn't exist in PostHog then we're safe to remove it. We never needed the debug functionality 👍

**Pull request merged**